src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/AbstractClassCouplingCheck.java

```diff
@@ -73,6 +75,12 @@
     private int max;
     /** Package of the file we check. */
     private String packageName;
+    /** User-configured regular expression to ignore classes. */
+    private String excludeClassesRegexp = "";
+    /**
+     * Regular expression pattern compiled from excludeClassesRegexp variable.
+     */
+    private Pattern excludeClassesRegexpPattern = CommonUtils.createPattern(excludeClassesRegexp);
```

Comment:
I know but I thought it will be useful for both classes as well, because they both extends common class and both work on metrics.